components/builder-api/src/server/resources/profile.rs

```diff
@@ -172,7 +173,7 @@ fn revoke_access_token(req: HttpRequest,
         Err(_) => return HttpResponse::new(StatusCode::UNPROCESSABLE_ENTITY),
     };
 
-    let account_id = match authorize_session(&req, None) {
+    let account_id = match authorize_session(&req, None, RoleAction::Read, None) {
```

Comment:
Should this be `Write`?

Reply:
No similar to above. A user needs to be able to revoke their own token.